test/functional/test_framework/wallet.py

```diff
@@ -414,11 +416,13 @@ def getnewdestination(address_type='bech32m'):
 
 def address_to_scriptpubkey(address):
     """Converts a given address to the corresponding output script (scriptPubKey)."""
+    version, payload = bech32_to_bytes(address)
```

Comment:
```suggestion
    version, payload = bech32_to_bytes(address)
```
@josibake  this will call will assert to false if the address is a base58 address.